src/languages/en.ts

```diff
@@ -2001,38 +2015,50 @@ export default {
-        taxRateChanged: 'dummy.violations.taxRateChanged',
-        taxRequired: 'dummy.violations.taxRequired',
+        allTagLevelsRequired: 'All tags required',
+        autoReportedRejectedExpense: ({rejectReason, rejectedBy}: ViolationsAutoReportedRejectedExpenseParams) => `${rejectedBy} rejected this expense with the comment "${rejectReason}"`,
+        billableExpense: 'Billable no longer valid',
+        cashExpenseWithNoReceipt: ({amount}: ViolationsCashExpenseWithNoReceiptParams) => `Receipt required over ${amount}`,
+        categoryOutOfPolicy: 'Category no longer valid',
+        conversionSurcharge: ({surcharge}: ViolationsConversionSurchargeParams) => `Applied ${surcharge}% conversion surcharge`,
+        customUnitOutOfPolicy: 'Unit no longer valid',
+        duplicatedTransaction: 'Potential duplicate',
+        fieldRequired: 'Report fields are required',
+        futureDate: 'Future date not allowed',
+        invoiceMarkup: ({invoiceMarkup}: ViolationsInvoiceMarkupParams) => `Marked up by ${invoiceMarkup}%`,
+        maxAge: ({maxAge}: ViolationsMaxAgeParams) => `Date older than ${maxAge} days`,
+        missingCategory: 'Missing category',
+        missingComment: 'Missing comment',
+        missingTag: ({tagName}: ViolationsMissingTagParams) => `Missing ${tagName ?? 'tag'}`,
+        modifiedAmount: 'Amount greater than scanned receipt',
+        modifiedDate: 'Date differs from scanned receipt',
+        nonExpensiworksExpense: 'Non-Expensiworks expense',
+        overAutoApprovalLimit: ({formattedLimitAmount}: ViolationsOverAutoApprovalLimitParams) => `Expense exceeds auto approval limit of ${formattedLimitAmount}`,
+        overCategoryLimit: ({categoryLimit}: ViolationsOverCategoryLimitParams) => `Amount over ${categoryLimit}/person category limit`,
+        overLimit: ({amount}: ViolationsOverLimitParams) => `Amount over ${amount}/person limit`,
+        overLimitAttendee: ({amount}: ViolationsOverLimitParams) => `Amount over ${amount}/person limit`,
+        perDayLimit: ({limit}: ViolationsPerDayLimitParams) => `Amount over daily ${limit}/person category limit`,
+        receiptNotSmartScanned: 'Receipt not verified. Please confirm accuracy.',
+        receiptRequired: ({amount, category}: ViolationsReceiptRequiredParams) => `Receipt required over ${amount} ${category ? ' category limit' : ''}`,
+        rter: ({brokenBankConnection, email, isAdmin, isTransactionOlderThan7Days, member}: ViolationsRterParams) => {
+            if (brokenBankConnection) {
+                return isAdmin
+                    ? `Can't auto-match receipt due to broken bank connection which ${email} needs to fix`
+                    : "Can't auto-match receipt due to broken bank connection which you need to fix";
+            }
+
+            if (!isTransactionOlderThan7Days) {
+                return isAdmin ? `Ask ${member} to mark as a cash or wait 7 days and try again` : 'Awaiting merge with card transaction.';
+            }
+
```

Comment:
Avoid adding blank lines that aren't preceded by comments
```suggestion
            if (!isTransactionOlderThan7Days) {
                return isAdmin ? `Ask ${member} to mark as a cash or wait 7 days and try again` : 'Awaiting merge with card transaction.';
            }
```